nailgun/entities.py

```diff
@@ -1868,8 +1869,8 @@ def path(self, which=None):
         ``super`` is called otherwise.
 
         """
-        if which == 'clone':
-            prefix = 'self' if which == 'clone' else 'base'
+        if which in ('clone', 'generate'):
```

Comment:
Just add a docstring for api_path of generate(just like clone) in this 'path' method and we are good to merge this PR.

Reply:
Fixed. Also, I think "clone" docstring was incorrect so I fixed that, too.